scipy/stats/setup.py

```diff
@@ -27,9 +32,17 @@ def configuration(parent_package='',top_path=None):
     config.add_extension('mvn',
         sources=['mvn.pyf','mvndst.f'],
     )
-        
-    # What do I add here to build Rectangle?
-    
+
+    # Build cython rectangle demo
+    config.add_data_files('rect.pxd')
```

Comment:
Must be added to data files and distributed because binaries reference this file!

Reply:
I recently ran into another situation where I needed to add a file. What exactly am I adding this file to? I understand that it makes it accessible, but I don't understand why certain files (e.g. `.py` files) seem to be accessible without adding them like this and others aren't.

Reply:
[Docs](https://numpy.org/devdocs/reference/distutils_guide.html) say that data files are copied by default to the package installation directory.  I guess this is similar to adding files to MANIFEST.in -- making sure they are available when the package is installed.  E.g.,

```
config.add_data_files('foo.dat')
```
adds the file to
```
<installation path of config.name package>/foo.dat
```

Reply:
I believe this is an issue of Python package installation conventions.  `.py` files are always added to distributions whereas other types of files are not.  `add_data_files` appears to be the `numpy.distutils` way of adding files to the `data_files` field of `setup`: https://docs.python.org/3/distutils/setupscript.html#distutils-additional-files